tests/kafkatest/services/verifiable_producer.py

```diff
@@ -73,24 +73,27 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
         self.produced_count = {}
-        self.prop_file = ""
+
+    def prop_file(self, idx):
+        prop_file = ""
+        self.security_config = self.kafka.security_config.client_config(self.prop_file)
```

Comment:
Also - this method gives the ability to construct different configs for different nodes - so it seems like the logic for setting `self.security_config` doesn't belong here since it is independent of the node, and would have unintuitive behavior if it did become dependent on the node?